app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -17,6 +17,19 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
+  const [notesList, setNotes] = useState(notes);
+  const [selectedNoteId, setSelectedNoteId] = useState();
+
+  const handleDelete = () => {
+    try {
+      const notes = notesList.filter(c => c.id !== selectedNoteId);
```

Comment:
Don't use random variable names like `c` etc which doesn't make any sense in the context. Here you are iterating through the notes. Thus each item is a `note`. NOT a `c`!